docs/docs/features/command-line-interface.md

```diff
@@ -30,6 +30,14 @@ NOTE: if you previously installed the legacy CLI, you will need to uninstall it
 npm uninstall -g immich
 ```
 
+## Installation (Docker)
+
+If npm is not available on your system you can try the docker version
+
+```bash
+docker run -it -v "$(pwd)":/import -v /tmp:/tmp ghcr.io/immich-app/immich-cli:latest
```

Comment:
What's the goal to mount the whole `/tmp` to the container ? Sounds like a security issue

Reply:
You might be right, maybe the whole /tmp is too much.

We need somewhere semi-persistent to store the auth.yml file between runs but maybe there's a better way

Reply:
Personally, I think that executing the cli with Docker is more a "throw-away execution":  With docker, I am sure that it will not impact the host, it just needs to access my assets (btw maybe add `:ro` to the `/import` volume) but in return I am ready to enter my API key for each use.

Reply:
yep, fix incoming and 90% done